config.dist.ini

```diff
@@ -0,0 +1,16 @@
+[firefox]
+windows=bin\webdrivers\gecko\geckodriver64.exe
```

Comment:
change to 
`windows=bin\webdrivers\gecko\geckodriver.exe`

so we are support 32bit as default